Tasks/ANT/task.json

```diff
@@ -13,7 +13,7 @@
     "version": {
         "Major": 1,
         "Minor": 0,
-        "Patch": 50
+        "Patch": 51
```

Comment:
We always seem to update patch version in all our tasks, wonder if we should update minor version and reset the patch version - e.g. 1.1.0. Not sure if it works though